### PR TITLE
feat: add proxy from env transport option

### PR DESCRIPTION
**Pull request by PG2000**
This PR adds the option to use a http proxy if set within environment variables

### CODE REVIEW

**Comment by PG2000** [comment]
/assign @jotadrilo 

Not sure if PR are picked up by you or i have to assign somebody.

File: internal/utils/utils.go [hunk 1]
```diff
@@ -27,7 +28,12 @@ const (
 
 var (
 	// UnixEpoch is the number of seconds that have elapsed since January 1, 1970
-	UnixEpoch = time.Unix(0, 0)
+	UnixEpoch          = time.Unix(0, 0)
+	DefaultClient      = &http.Client{Transport: &http.Transport{Proxy: http.ProxyFromEnvironment}}
+	InsecureHttpClient = &http.Client{Transport: &http.Transport{
```
Comment (jotadrilo) [inline internal/utils/utils.go:33]: I would remove Http for simplicity
```suggestion
	InsecureClient = &http.Client{Transport: &http.Transport{
```

Comment (PG2000) [reply internal/utils/utils.go]: @jotadrilo ok. Done that

**Review by jotadrilo — APPROVED** [review]
LGTM!

**Comment by jotadrilo** [comment]
Please take a look at the tests results!

**Comment by PG2000** [comment]
> Please take a look at the test results!

mhh doesn't happen on my machine.
```
GO111MODULE=on go test ./...
?   	github.com/bitnami-labs/charts-syncer	[no test files]
ok  	github.com/bitnami-labs/charts-syncer/api	0.260s
?   	github.com/bitnami-labs/charts-syncer/cmd	[no test files]
?   	github.com/bitnami-labs/charts-syncer/internal/cache	[no test files]
ok  	github.com/bitnami-labs/charts-syncer/internal/chart	1.477s
ok  	github.com/bitnami-labs/charts-syncer/internal/config	0.359s
?   	github.com/bitnami-labs/charts-syncer/internal/indexer	[no test files]
?   	github.com/bitnami-labs/charts-syncer/internal/indexer/api	[no test files]
ok  	github.com/bitnami-labs/charts-syncer/internal/utils	1.901s
?   	github.com/bitnami-labs/charts-syncer/pkg/client	[no test files]
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/intermediate	1.646s
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/repo	3.605s
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/repo/chartmuseum	1.920s
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/repo/harbor	2.075s
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/repo/helmclassic	2.145s
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/repo/local	2.375s
ok  	github.com/bitnami-labs/charts-syncer/pkg/client/repo/oci	1.371s
?   	github.com/bitnami-labs/charts-syncer/pkg/client/types	[no test files]
ok  	github.com/bitnami-labs/charts-syncer/pkg/syncer	1.925s
```

Can i retrigger the pipeline somehow?

**Pull request merged**